models/iaf_psc_alpha_multisynapse.cpp

```diff
@@ -177,18 +197,22 @@ void
 iaf_psc_alpha_multisynapse::State_::get( DictionaryDatum& d,
   const Parameters_& p ) const
 {
-  def< double >( d, names::V_m, y3_ + p.E_L_ ); // Membrane potential
+  def< double >( d, names::V_m, V_m_ + p.E_L_ ); // Membrane potential
 }
 
 void
 iaf_psc_alpha_multisynapse::State_::set( const DictionaryDatum& d,
   const Parameters_& p,
   const double delta_EL )
 {
-  if ( updateValue< double >( d, names::V_m, y3_ ) )
-    y3_ -= p.E_L_;
+  if ( updateValue< double >( d, names::V_m, V_m_ ) )
```

Comment:
I know you already did it before, but I think the code would be more clear if you write again a short comment explaining why this `if` condition is used.